packages/web3-providers-ipc/test/unit/check_implementation.test.ts

```diff
@@ -97,23 +97,6 @@ describe('IPCProvider', () => {
 			expect(end).toHaveBeenCalled();
 		});
 
-		it('connection error', async () => {
```

Comment:
removed connection error test, mock implementation doesn't seem to emit error for this one unit test so i added an integration test to substitute it
https://github.com/web3/web3.js/pull/6823/files#diff-ea6da20dfb39c75b95f9a7cb4cf0e3f9d1e353fe0aab69f28ef83497e86ff556R140